action.yml

```diff
@@ -24,5 +24,22 @@ inputs:
     default: main
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: "composite"
+  steps:
+    - name: Set up pixi
+      uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659  # v0.8.1
+      with:
+        cache: true
+        cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
```

Comment:
I'm not fully clear on how `github.ref_name` works when the action is being used by other workflows, so this might all be wrong. (Here I'm just copying from the https://github.com/prefix-dev/setup-pixi README).

Reply:
This still holds true, but worst case scenario we cache for some and not others and I think that's okay.